.maintain/monitoring/README.md

```diff
@@ -3,26 +3,32 @@
 ## Requirements
 
 1. `forest` node running locally
-2. `docker` & `docker-compose` must be available in the `$PATH`
+2. `docker` & `docker-compose` must be available in the `$PATH` (alternatively `podman` and `podman-compose` can be used)
```

Comment:
Have you checked that `podman` works out of the box? I had some issues with it in the past, like having to add some additional settings to the compose file to make it work.

Reply:
Yes I use `podman` locally and it works